tests/e2e/cucumber/features/smoke/copyMove.feature

```diff
@@ -56,4 +66,40 @@ Feature: Copy
     And "Alice" moves the following resource using drag-drop-breadcrumb
       | resource | to          |
       | Sub2     | PARENTCopy4 |
+
+    And "Alice" opens the "files" app
+    And "Alice" opens folder "PARENT"
+    And "Alice" copies the following resources to "PARENT/Sub1" at once using dropdown-menu
+      | resource        |
+      | fileToCopy1.txt |
+      | fileToCopy2.txt |
```

Comment:
can we do it with more resources? doing copy/move between folders that would reduce number of files to create

Reply:
Yes, I agree. You are moving 2 files at a time. It is pretty few for checking
How said Sajan, I suggest to move all files `fileToCopyN.txt` to folder sub1, after that from sub1 to sub2 ...